noir_stdlib/src/convert.nr

```diff
@@ -175,15 +175,15 @@ comptime fn generate_as_primitive_impls(_: FunctionDefinition) -> Quoted {
     let mut impls = &[];
     for type1 in types {
         for type2 in types {
-            impls = impls.push_back(
-                quote {
+            let trait_impl = quote {
                 impl AsPrimitive<$type1> for $type2 {
                     fn as_(self) -> $type1 {
                         self as $type1
                     }
                 }
-            },
-            );
+            };
+
+            impls = impls.push_back(trait_impl);
```

Comment:
This is listed as a workaround. Do we have an issue for the issue this is working around?

Reply:
Kind of. It's this bug: https://github.com/noir-lang/noir/pull/8740
I'll create an issue for this.